xml/smb_native_lm.xml

```diff
@@ -41,5 +41,21 @@
     <param pos="0" name="service.vendor" value="Samba"/>
     <param pos="0" name="service.product" value="Samba"/>
     <param pos="1" name="service.version"/>
+    <param pos="0" name="service.certainty" value="1.00"/>
   </fingerprint>
+
+  <!-- Windows -->
+  <fingerprint pattern="^(Windows [\W*\w*]*)$">
```

Comment:
We should be very careful here when "recog-ifying" any remaining parts of Nexpose.  What ever ends up in Recog should be as generic and non-product (Nexpose) specific as possible.